simulator/Core.ts

```diff
@@ -67,6 +67,8 @@ export class Core implements ICore {
 
         address = this.wrap(address);
 
+        this.triggerEvent(task, address, CoreAccessType.execute);
```

Comment:
Cool, I'll have to think of a nice way to pass these to the canvas and call efficient re-renders